Robot2018/src/org/usfirst/frc/team199/Robot2018/subsystems/DrivetrainInterface.java

```diff
@@ -126,6 +128,11 @@
 	public void shiftGearSolenoidOff();
 
 	/**
+	 * Returns the gyroscope
+	 * @return the gyroscope
```

Comment:
again here, this documentation comment is redundant

Reply:
I copied the code from Drivetrain.  :)